docs/developers/contributing.rst

```diff
@@ -33,7 +33,7 @@ a bug report:
     # etc.
     ```
 
-2. An explanation of why the current behaviour is wrong/not desired, and what you
+2. An explanation of why the current behavior is wrong/not desired, and what you
```

Comment:
Boo for 🇺🇸 spelling 😉 . My serious thought here (not for this PR), is can we configure codespell to enforce EN-US instead of EN-GB?